api/tests/Api/ReviewsTest.php

```diff
@@ -7,11 +7,14 @@
 use ApiPlatform\Symfony\Bundle\Test\ApiTestCase;
 use ApiPlatform\Symfony\Bundle\Test\Client;
 use App\Entity\Book;
+use App\Tests\Fixtures\Story\DefaultReviewsStory;
 use Symfony\Component\HttpFoundation\Response;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
 
 final class ReviewsTest extends ApiTestCase
 {
-    use RefreshDatabaseTrait;
+    use ResetDatabase, Factories;
```

Comment:
PSR-12 recommends to include one-per-line for use traits: https://www.php-fig.org/psr/psr-12/#42-using-traits